packages/core/src/constants/type/Config.ts

```diff
@@ -57,6 +57,10 @@ export type FullConfig = {
   localStorage: {
     transactionPath: string
   }
+  /**
+   * Gas limit percentage number.
```

Comment:
```suggestion
   * If set, adds an additional buffer of gas limit to estimated gas limit before sending a transaction.
   * Useful if a gas limit of a transaction can be different depending on the state of the blockchain.
   * Gas estimation can be not accurate because the state of the blockchain can change between the time of estimation and the time of transaction mining.
```